platform/remarkable/README_remarkable.txt

```diff
@@ -13,12 +13,35 @@ with a USB cable.
    ssh root@<your-remarkable>
 
    unzip <koreader-package>.zip
+
+# Launching using the included launcher
+
+koreader includes a very simple launcher called `button-listen` which starts
```

Comment:
```suggestion
KOReader includes a very simple launcher called `button-listen`, which starts
```

Reply:
Ah yes, I've corrected the captialisation of koreader->KOReader throughout